doc/releasenotes/v0.5.0.rst

```diff
@@ -90,6 +90,7 @@ Python API Changes
 ..
   Describe any changes to the API
 
+- The test suite has been overhauled and moved to the `tests/` directory. The test suite is no longer part of the `saltproc` package.
```

Comment:
```suggestion
- The test suite has been overhauled and moved to the `tests/` directory.
```

Reply:
This should stay as before, the test suite was explicitly included in the `saltproc` package as `saltproc.tests`